measures/2021/measures-data.json

```diff
@@ -18861,11 +18861,11 @@
     "overallAlgorithm": "weightedAverage",
     "strata": [
       {
-        "name": "mpiStudy",
+        "name": "attentuationCorrectn",
```

Comment:
Can you confirm the spelling on this one.

Reply:
It is spelled as was provided by Michelle. Needed to be shortened to fit the 20 character limit on strata names